manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.locale.en-US.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: FACEITLtd.FACEITClient
+PackageVersion: 1.30.0
+PackageLocale: en-US
+Publisher: FACEIT Ltd.
+PackageName: FACEIT Client

```

```suggestion
PackageName: FACEIT
```
Wops Missed this one